google_play_scraper/constants/element.py

```diff
@@ -151,6 +146,37 @@ class ElementSpecs:
     }
 
     Permission_Type = ElementSpec(None, [0])
+
     Permission_List = ElementSpec(
         None, [2], lambda container: sorted([item[1] for item in container])
     )
+
+    DataSafety = {
+        "data_collected": ElementSpec(3, [1, 2, 137, 4, 1, 0],
+            lambda collection: {ElementSpec(None, [0, 1]).extract_content(collection[i]): ElementSpec(None, [4],
+                lambda entrys: [
+                    {
+                        'name': ElementSpec(None, [0]).extract_content(entrys[j]),
+                        'optional': ElementSpec(None, [1]).extract_content(entrys[j]),
+                        'usage': ElementSpec(None, [2], None, None).extract_content(entrys[j])
+                    } for j in range(0, len(entrys))] 
+                ).extract_content(collection[i]) for i in range(0, len(collection))},
+            ),
+        "data_shared": ElementSpec(3, [1, 2, 137, 4, 0, 0],
+            lambda collection: {ElementSpec(None, [0, 1]).extract_content(collection[i]): ElementSpec(None, [4],
+                lambda entrys: [
+                    {
+                        'name': ElementSpec(None, [0]).extract_content(entrys[j]),
+                        'optional': ElementSpec(None, [1]).extract_content(entrys[j]),
+                        'usage': ElementSpec(None, [2], None, None).extract_content(entrys[j])
+                    } for j in range(0, len(entrys))] 
+                ).extract_content(collection[i]) for i in range(0, len(collection))},
+            ),
+        "security_practices": ElementSpec(3, [1, 2, 137, 9, 2],
+            lambda container: [
+                {
+                    'name': ElementSpec(None, [i, 1]).extract_content(container),
+                    'description': ElementSpec(None, [i, 2, 1]).extract_content(container)
+                } for i in range(0, len(container))
+            ])
+    }
```

Comment:
@JoMingyu i don't like this double nested lamda function at all, especially since it is identical for data_collected and data_shared. do you have an idea? i was thinking of just defining the elementspecs for a single entry here and replacing the outer lamda function with a classic loop in data_safety.py, similar to what we do for the reviews.

Reply:
Sorry for the very slow reply. I didn't turn on notifications for mention, and the PR was Draft, so I left it alone. First, if you are interested in becoming a maintainer, please email me. The PRs you've contributed so far have been very effective, so I'm ready to welcome.

Back to the point, I'm not satisfied with the double nested structure. At the time of developing the initial version, it seems that the field of view was not wide. It's okay to write a PR as you said. I am open to large-scale changes.

Reply:
@kluhan mention for notification